### PR TITLE
Update extending.rst

### PR DESCRIPTION
**Pull request by cr0hn**
Add an alternative way to load full user data

### CODE REVIEW

File: docs/how_to_guides/extending.rst [hunk 1]
```diff
@@ -39,10 +39,39 @@ good example of this is loading user data from a database,
...
+    app = Quart(__name__)
+
+    @app.route("/")
+    @login_required
+    async def index():
+        return current_user.email
+
+    auth_manager.init_app(app)
+    
+    @app.before_request
```
Comment (pgjones) [inline docs/how_to_guides/extending.rst:75]: I'd add `@app.before_websocket` as well

Comment (cr0hn) [reply docs/how_to_guides/extending.rst:75]: Nice!

Comment (pgjones) [reply docs/how_to_guides/extending.rst:75]: Could you change the PR?

Comment (cr0hn) [reply docs/how_to_guides/extending.rst:75]: Sorry, I still have not experienced Quart-auth and WebSockets. I don't know how to do it

**Comment by pgjones** [comment]
Thanks, merged as 865214679d2f5e54820c48d6404b8201de4e492f

**Pull request closed without merging**